src/gui/shader.cpp

```diff
@@ -1,13 +1,16 @@
 #include "shader.hpp"
 
+#include "../logging.hpp"
+
 #include <GL/glew.h>
 
 #include <filesystem>
 #include <fstream>
-#include <iostream>
 #include <string>
 #include <vector>
 
+static quill::Logger* logger = logging::get_logger("gui.shaders");
```

Comment:
Is this the file name? because it's not.

Reply:
![image](https://user-images.githubusercontent.com/76668284/208224342-56a2bbce-0c9f-400a-895e-0794cb7c3f1d.png)

Reply:
`shaders`

### ers

### S

Reply:
Yes

Because this is responsible for loading all the shader**S**